src/components/AudioDetails/Related.vue

```diff
@@ -3,22 +3,56 @@
     <h4 class="b-header mb-6">
       {{ $t('audio-details.related-audios') }}
     </h4>
-    <AudioTrack
-      v-for="audio in relatedAudios"
-      :key="audio.id"
-      :audio="audio"
-      :is-compact="true"
-      layout="row"
-      class="mb-12"
-    />
+    <template v-if="!$fetchState.error">
+      <AudioTrack
+        v-for="audio in audios"
+        :key="audio.id"
+        :audio="audio"
+        layout="row"
+        size="m"
+        class="mb-12"
+      />
+      <LoadingIcon v-show="$fetchState.pending" />
+    </template>
+    <p v-show="!!$fetchState.error">
```

Comment:
`fetchState.error` can be null, if there is no error, or be an object containing the error message. Double negation is to convert this to Boolean: if it's null, it would be converted to false (`null`, `!null` = `true`, `!!null` = `!true`), and if it's an object, it would be converted to true (`{error: ''}`, `!{error: ''}` = `false`, !!{error: ''}` = `true`).